tests/Aspire.MySqlConnector.Tests/AspireMySqlConnectorExtensionsTests.cs

```diff
@@ -37,7 +44,9 @@ public void ReadsFromConnectionStringsCorrectly(bool useKeyed)
             host.Services.GetRequiredKeyedService<MySqlDataSource>("mysql") :
             host.Services.GetRequiredService<MySqlDataSource>();
 
-        Assert.Equal(ConnectionString, dataSource.ConnectionString);
+        string actualConnectionString = dataSource.ConnectionString;
+        string expectedConnectionString = new MySqlConnectionStringBuilder(ConnectionString).ConnectionString;
```

Comment:
```suggestion
        string expectedConnectionString = ConnectionString;
```

I think this should be unnecessary? The `ConnectionString` constant is already in the "normalized" form so this round-tripping through `MySqlConnectionStringBuilder` will be a no-op.